akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/SideEffect.scala

```diff
@@ -14,9 +14,10 @@ import akka.japi.function
  * Persist, none and unhandled are main effects. Then any number of
  * call backs can be added to these effects with `andThen`.
  *
- * Not for user extension
+ * INTERNAL API
  */
-sealed abstract class SideEffect[State]
+@InternalApi
+private[akka] sealed abstract class SideEffect[State]
 
```

Comment:
internal API markers on `object SideEffect`, and it looks like `apply`, `create` and `stop` are not used there and can be removed